wiki/Reporting_Bad_Behaviour/Handling_Cheating/en.md

```diff
@@ -1,74 +1,74 @@
 
-Prove that I/they were cheating! Tell us how you know!
-------------------------------------------------------
 
-The staff cannot say on how can they prove. Giving away the countermeasures would make them useless in further detecting cheaters.
+## Prove that I/they were cheating! Tell us how you know!
+The team cannot say on how can they prove. Giving away the countermeasures would make them useless in further detecting cheaters.
 
-I'll make a video to prove my innocence!
-----------------------------------------
 
+## I'll make a video to prove my innocence!
 Sorry but if you have been banned, they are already 100% sure you are cheating. A video will not help.
 
-I found someone who is definitely cheating. I can tell by their replay/score! What should I do?
------------------------------------------------------------------------------------------------
 
-Stay calm.
+## I found someone who is definitely cheating. I can tell by their replay/score! What should I do?
+Stay calm and **don't discuss their play in public chat**.
+Don't give them undue attention or ask others to spectate them. There are three ways to report users.
 
-**DON'T discuss their play in public chat**.
+### User Profile
+On the user's profile, a `Report User` button will appear when you move your cursor to their avatar. Write any information that can prove that the user is doing misbehaviour.
 
-Don't give them undue attention or ask others to spectate them.
+*Take note of the selected game mode before reporting*
 
-Click the "Report" link at the right side of their score on the beatmap info (website)
+### Beatmap Page
+In beatmap pages, you can report a user in the top leaderboards by clicking the `Report` button on the far right of their score.
 
-Use !report in in-game chat to send a message to moderators.
+### In-game chat
+Use the `!report` command in any public channel to inform the chat moderators. Other players will not see your reports in chat.
 
-`Example: !report `<Username>` is suspected to be a cheater.`
+      Format: !report <username> <reason>
+     Example: !report this_user suspected cheating
 
-Your help in spotting foul play is much appreciated. You can also send an email directly to osu@ppy.sh for a relatively fast response.
+Your help in spotting foul play is much appreciated. You can also send an email directly to [osu@ppy.sh](mailto:osu@ppy.sh) for a relatively fast response.
 
-Is using specialised hardware counted as cheating?
---------------------------------------------------
 
+## Is using specialized hardware counted as cheating?
 Peripherals such as tablets and touchscreens are fine. Anything which decreases the skill required to play is not. This means that using multiple PCs, input splitters or anything of that kind is considered a no-no.
```

Comment:
It might be slightly biased because I'm a taiko guy, but probably should add stuff like

- Second monitor (HDFL playing without having to memorize)
- Double keyboard or others (mashing through 1/8 streams with nearly perfect accuracy)

Reply:
*Anything which decreases the skill required to play is not* so those are included in that phrase. Common sense shall prevail. ;)

Reply:
For the sake of clarity, you might want to add something along of the lines of "cancels/defeats the effect of game modifiers"